CHANGES.rst

```diff
@@ -28,6 +28,12 @@ extract_1d
 - Fixed a bug in the calling of optional MIRI MRS 1d residual fringe
   correction that could cause defringing to fail in some cases. [#8180]
 
+photom
+------
+
+- Fixed a bug in multislit data that did not update the bunit_data and
```

Comment:
Should update this info now to indicate that the values in the "primary" part of the data model are blanked out (set to None), rather than getting updated with a copy of what's in the slit instances.